jaxopt/_src/perturbations.py

```diff
@@ -45,7 +45,8 @@ def log_prob(self, inputs: jax.Array) -> jax.Array:
 def make_perturbed_argmax(argmax_fun: Callable[[jax.Array], jax.Array],
                           num_samples: int = 1000,
                           sigma: float = 0.1,
-                          noise=Gumbel()):
+                          noise=Gumbel(),
+                          reduce_variance: bool = False):
```

Comment:
Hi @q-berthet,

Thank you for the corrections (I have just corrected them in the new commit).

You are completely right, `control_variate` is a much better name, instead of `reduce_variance`. Indeed for large `sigma`, there are cases in which no control variate can give better estimations of the true jacobian. I also changed the docstring for the function to explain what is going on a bit clearer. Cheers!
Lawrence